lib/pharos/up_command.rb

```diff
@@ -108,7 +108,11 @@ def configure(config)
       craft_time = Time.now - start_time
       puts pastel.green("==> Cluster has been crafted! (took #{humanize_duration(craft_time.to_i)})")
       puts "    You can connect to the cluster with kubectl using:"
-      puts "    export KUBECONFIG=~/.pharos/#{manager.sorted_master_hosts.first.api_address}"
+      if config.cluster&.kube_config&.path
+        puts "    export KUBECONFIG=#{config.cluster.kube_config.path}"
+      else
```

Comment:
IMHO this does not make sense, we should just default to the old config path if user has not defined path in cluster.yml. Would make code easier to read (no if/else)